load.php

```diff
@@ -288,16 +297,46 @@ function perflab_is_standalone_plugin_loaded( $module ) {
 }
 
 /**
- * Gets the standalone plugin constants used for each module / plugin.
+ * Gets the standalone plugin constants used for each module with a standalone plugin.
  *
  * @since 2.2.0
+ * @deprecated n.e.x.t
  *
  * @return array Map of module path to version constant used.
  */
 function perflab_get_standalone_plugins_constants() {
+	_deprecated_function( __FUNCTION__, 'Performance Lab n.e.x.t', "perflab_get_standalone_plugin_version_constants( 'modules' )" );
```

Comment:
The reference to `n.e.x.t` will not change using the command `npm run since -- -r 2.9.0`. So, make sure to update it manually or we can update it now before the merge. What do you think?